lib/manageiq/providers/workflows/builtin_methods.rb

```diff
@@ -0,0 +1,16 @@
+module ManageIQ
+  module Providers
+    module Workflows
+      class BuiltinMethods < BasicObject
+        def self.email(params = {}, _secrets = {}, _context = {})
+          options = params.slice("To", "From", "Subject", "Cc", "Bcc", "Body", "Attachment").transform_keys { |k| k.downcase.to_sym }
+
+          require "generic_mailer"
```

Comment:
I think this should not be necessary (I'm concerned cause this can break things like zeitwerk)?  If it's needed, perhaps this is happening because of the BasicObject. In that case maybe something like `Object.const_get("GenericMailer")`?

Reply:
I had to add this after `< BasicObject`

Reply:
Or maybe we just get rid BasicObject and solve that problem differently.

Reply:
```
/home/grare/adam/src/manageiq/manageiq-providers-workflows/lib/manageiq/providers/workflows/builtin_methods.rb:9:in `email': uninitialized constant ManageIQ::Providers::Workflows::BuiltinMethods::Object (NameError)

          miq_task = Object.const_get("GenericMailer").deliver_task(:generic_notification, options)
                     ^^^^^^
Did you mean?  ObjectSpace
```

Reply:
oh, maybe `::Object`

Reply:
Seems like `::Object.const_get` does work

Reply:
Ha yeah, still seems odd I'm tempted to whitelist the methods and drop the BasicObject